x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -167,10 +171,18 @@ export const exportFirstRule = () => {
 export const filterBySearchTerm = (term: string) => {
   cy.log(`Filter rules by search term: "${term}"`);
   cy.get(RULE_SEARCH_FIELD)
-    .type(term, { waitForAnimations: true })
+    .type(term, { force: true })
     .trigger('search', { waitForAnimations: true });
 };
 
+export const filterByTags = (tags: string[]) => {
+  cy.get(RULES_TAGS_FILTER_BTN).click();
```

Comment:
Do any of these `clicks` need a force true to avoid potential flakiness?

Reply:
It's a good question. We discussed it inside our area but don't have a clear decision yet. It's clear that we wanna avoid flakiness as much as possible while forcing all UI manipulations isn't a good approach either. I'd like to avoid `force: true` in added tests and will keep an eye on it.

Flakiness is rather a wide meaning. If a test fails time to time in sequential runs it's flaky for sure. Another situation when a test fails due to some changes so it either means test is flaky or there is something we need to pay attention to. For example we [had to disable](https://github.com/elastic/kibana/pull/151244) some e2e tests because they suddenly started failing in `main` since `force: true` was missing but it helped to find an UI problem on our side which was exposed by a new EUI version.